airflow/www_rbac/views.py

```diff
@@ -1066,6 +1098,17 @@ def _clear_dag_tis(self, dag, start_date, end_date, origin,
             flash("No task instances to clear", 'error')
             response = redirect(origin)
         else:
+            if event_name is not None:
```

Comment:
is `event_name` ever None? Afaict it is not.

Reply:
its not but since its a kwarg, i feel like its nice to have the check so we dont log something with none as the event name

Reply:
if default doesnt make sense let's make it an kwarg.